gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
<p>I don't think the prompt is needed in two places, unless you can't read the code in BQ studio notebooks or something</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Yeah, you can't read it in BQ Studio (I added this for my demo). I'll remove it in this version of the notebook.</p>